app/pages/project/home/project-home.jsx

```diff
@@ -114,9 +114,8 @@ const ProjectHomePage = (props) => {
                     controls={false}
                     type={type}
                     format={format}
-                    height={240}
                     src={src}
-                    width={600}
+                    width={400}
```

Comment:
Because of the container classname `project-home-page__container`, I think the Talk Subjects Preview section is intentionally a full-viewport-width section. The change to Thumbnail's width here creates breaks between the thumbnails in the full width design on a 21in monitor:

Original:
![original](https://user-images.githubusercontent.com/23665803/212366677-11db21c1-5a7e-4b36-b7d3-88c9c8b34e41.jpg)


This PR:
![change](https://user-images.githubusercontent.com/23665803/212366706-28335035-65c3-478b-aa25-069741e9851f.jpg)

I recommend reverting to 600 width, or a dimension that's large enough for a typical iMac desktop screen.

Reply:
D’oh! Will do.

Reply:
Done! Should be updated on the staged branch shortly.